plugins/modules/site_workflow_manager.py

```diff
@@ -730,17 +939,40 @@ def get_have(self, config):
         site_exists = False
         current_site = None
         have = {}
+        try:
+            if self.get_ccc_version_as_integer() >= self.version_2_3_7_6 and self.config[0]['site']['type'] == "bulk":
+                (site_exists, current_site) = self.site_exists()
+                self.log("Current Site details (have): {0}".format(str(current_site)), "DEBUG")
+                if site_exists:
+                    have["site_exists"] = site_exists
+                    have["current_site"] = current_site
+                    self.log(current_site)
```

Comment:
Add some message before directly printing the variable value.

Reply:
Added